apps/crn-frontend/src/discover/Routes.tsx

```diff
@@ -16,44 +16,67 @@ const loadWorkingGroups = () =>
 const loadAbout = () =>
   import(/* webpackChunkName: "discover-about" */ './About');
 
+const loadTutorial = () =>
+  import(
+    /* webpackChunkName: "tutorials-details-page" */ './tutorials/Tutorial'
+  );
+
 const Guides = lazy(loadGuides);
 const Tutorials = lazy(loadTutorials);
 const WorkingGroups = lazy(loadWorkingGroups);
 const About = lazy(loadAbout);
+const TutorialDetailsPage = lazy(loadTutorial);
 
 const Discover: FC<Record<string, never>> = () => {
   useEffect(() => {
-    loadGuides().then(loadTutorials).then(loadWorkingGroups).then(loadAbout);
+    loadGuides()
+      .then(loadTutorials)
```

Comment:
I am thinking this naming is now confusing as there is already a mistake here. Tutorials should probablly be moved into the tutorials folder and be called TutorialList like in Network

Reply:
Nice! It looks a lot better now :) It was committed here https://github.com/yldio/asap-hub/pull/2143/commits/91475a81651ecadd06c37c2c5bdc11d42e01cb4b